plugins/provisioning-terraform-aws-deployment-ecs/src/static/ecs-template.tf

```diff
@@ -0,0 +1,195 @@
+module "${{ ECS_CLUSTER_MODULE_NAME }}" {
+  source  = "terraform-aws-modules/ecs/aws//modules/cluster"
+  version = "5.2.2"
+
+  cluster_name = "${{ CLUSTER_NAME }}"
+
+  ${{ CLUSTER_CAPACITY_PROVIDER }}
+}
+
+module "${{ ECS_SERVICE_MODULE_NAME }}" {
+  source  = "terraform-aws-modules/ecs/aws//modules/service"
+  version = "5.2.2"
+
+  name        = "${{ SERVICE_NAME }}"
+  cluster_arn = module.${{ ECS_CLUSTER_MODULE_NAME }}.arn
+
+  cpu    = 1024
+  memory = 4096
+
+  container_definitions = {
+    ("${{ SERVICE_NAME }}") = {
+      essential = true
+      cpu       = 512
+      memory    = 1024
```

Comment:
Do these need to be aligned with these: https://github.com/amplication/plugins/pull/223/files#diff-c0c1f29735586f276d87541dac692490c30d486d445f98bc017746ebe3a18d09R17 ?

Reply:
https://github.com/amplication/amplication-cluster-configuration/issues/111